tests/test_plugins/test_plugin/server.py

```diff
@@ -22,31 +22,33 @@
 from girder.api.describe import Description, describeRoute
 from girder.api.rest import boundHandler, rawResponse, Resource, setResponseHeader
 from girder.api.v1.collection import Collection
+from girder.constants import TokenScope
 from girder.utility.server import staticFile
 
 
-@access.public
+@access.user(scope=TokenScope.ANONYMOUS_SESSION)
```

Comment:
I don't follow the rationale of changing all of the public access points in this test (in the context of the bound handler decorator change) to require anonymous session scope.